src/common/transformer/test/test.server.js

```diff
@@ -1,9 +1,20 @@
 const expect = require('chai').expect;
-const markoCompiler = require('marko/compiler');
 const prettyPrint = require('marko-prettyprint').prettyPrintAST;
-const CompileContext = require('marko/compiler/CompileContext');
-const Builder = require('marko/compiler/Builder');
+const markoCompiler = require('marko/compiler');
 const transformer = require('../');
+let CompileContext;
+let Builder;
+
+
```

Comment:
My preference for no double lines. 🤷🏻‍♂️

Reply:
FYI I opened an issue here to see if we want to apply that style rule globally   
https://github.com/eBay/eslint-config-ebay/issues/10